.travis.yml

```diff
@@ -16,7 +16,7 @@
 
 language: cpp
 dist: trusty
-sudo: false
+sudo: required
 
```

Comment:
I missed that now Travis has deprecated the  container-based Trusty infrastructure, which required `sudo: false`.
Anyway, you should be able to install the needed Qt5 stuff as Felix did [here](https://github.com/Qucs/qucs/pull/898/files#diff-354f30a63fb0907d4ad57269548329e3R51) in his PR.

Reply:
That seems like a better idea, thanks